types/dec_coin_test.go

```diff
@@ -287,43 +292,50 @@ func TestSortDecCoins(t *testing.T) {
 	}
 
 	cases := []struct {
+		name          string
 		coins         DecCoins
 		before, after bool // valid before/after sort
 	}{
-		{good, true, true},
-		{empty, false, false},
-		{badSort1, false, true},
-		{badSort2, false, true},
-		{badAmt, false, false},
-		{dup, false, false},
+		{"valid coins", good, true, true},
+		{"empty coins", empty, false, false},
+		{"unsorted coins (1)", badSort1, false, true},
+		{"unsorted coins (2)", badSort2, false, true},
+		{"zero amount coins", badAmt, false, false},
+		{"duplicate coins", dup, false, false},
 	}
 
-	for tcIndex, tc := range cases {
-		require.Equal(t, tc.before, tc.coins.IsValid(), "coin validity is incorrect before sorting, tc #%d", tcIndex)
+	for _, tc := range cases {
+		require.Equal(t, tc.before, tc.coins.IsValid(), "coin validity is incorrect before sorting; %s", tc.name)
 		tc.coins.Sort()
-		require.Equal(t, tc.after, tc.coins.IsValid(), "coin validity is incorrect after sorting, tc #%d", tcIndex)
+		require.Equal(t, tc.after, tc.coins.IsValid(), "coin validity is incorrect after sorting;  %s", tc.name)
 	}
 }
 
-func TestDecCoinsIsValid(t *testing.T) {
+func TestDecCoinsValidate(t *testing.T) {
```

Comment:
it'd be nice if there was a large set of denom strings both `DecCoins` and `Coins` could both use for testing to ensure equal test cases - just a suggestion though